src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -227,6 +230,7 @@ export default class FilterableMultiSelect extends React.Component {
                 disabled={disabled}
                 invalid={invalid}
                 invalidText={invalidText}
+                tabIndex="-1"
```

Comment:
Why do we need the `-1` on the ListBox root?

Reply:
Good call